java-generator/pom.xml

```diff
@@ -45,7 +45,7 @@
     <groovy-api.version>2.4.21</groovy-api.version>
     <javax.inject.version>1</javax.inject.version>
     <commons-io.version>2.13.0</commons-io.version>
-    <guava.version>32.0.1-jre</guava.version>
+    <guava.version>32.1.0-jre</guava.version>
```

Comment:
I see guava being referenced in java-generator/it pom:

https://github.com/fabric8io/kubernetes-client/blob/449d06bc447f2b621149b20312235d2d28460873/java-generator/it/pom.xml#L107-L110

However, I'm not able to find any additional references to guava in this repository. Even if I remove this dependency all java generator tests seem to pass.

I think we should try to get rid of this dependency.